bedrock/pocket/templates/pocket/platform-pages.html

```diff
@@ -26,6 +26,12 @@
           <h1>{{ self.page_title() }}</h1>
           <p>{{ self.page_desc() }}</p>
           <a href="{{ self.page_url() }}" class="mzp-c-button" target="_blank" rel="external noopener">{{ self.page_cta() }}</a>
+          {% block bookmarklet %}
+          <p>
+            <a class="bookmarklet" href="https://getpocket.com/bookmarklets/">Install the bookmarklet &raquo;</a>
```

Comment:
Issue (blocking): string should be translatable?

Bonus points: seeing as it's a new string, if we wanted it to only show up on locales when it gets translated, we could use `ftl_has_messages()`: https://bedrock.readthedocs.io/en/latest/l10n.html#the-ftl-has-messages-helper-function

Reply:
Suggestion (non-blocking): we should try and keep purely decorative characters like `»` out of page copy, as they will get read aloud by screen readers this way. Instead, suggest changing this to a CSS pseudo element:

```scss
.bookmarklet {
    // existing styles 

    &:after {
        content: " »";
    }
}
```

Reply:
Question (non-blocking): if `/bookmarklets/` redirects directly to `/add/`, any reason not to link directly to it here with `url('pocket.add')`?

Reply:
@alexgibson Thanks for the reminder about conditional showing of the link - will sort.

as for /bookmarklet/ vs direct to /add - that redirection happens somewhere on the Pocket side and I don't want to move any wiring that we don't own in case they want to do something different in the future (or have some metrics linked to the redirection from /bookmarklet/, etc). Once we're live with this on Bedrock, I thikn we can start to remove unncessary redirects, though

Reply:
@agibson Polished up and ready for re-review 👍